irods/test/data_obj_test.py

```diff
@@ -1317,7 +1323,7 @@ def test_obj_put_get(self):
 
         # make random 16M binary file
```

Comment:
update comment

Reply:
I have updated the comment and added two more comments for the tests.

Reply:
perfect - thanks!